mov_cli/media/quality.py

```diff
@@ -19,4 +29,33 @@ def __init__(self, pixel: int) -> None:
 
     def apply_p(self) -> str:
         """Returns that enum but with an ending p."""
-        return f"{self.value}p"
\ No newline at end of file
+        return f"{self.value}p"
+
+def get_quality(url: str | Path) -> Quality | None:
+    if not shutil.which("ffprobe"):
+        return None
+
+    args = [
+        "ffprobe", 
+        "-v", 
+        "error", 
+        "-select_streams", 
+        "v", 
+        "-show_entries", 
+        "stream=width,height", 
+        "-of",
+        "json",
+        url
+    ]
+
+    out = str(subprocess.check_output(args), "utf-8")
+
+    stream = json.loads(out).get("streams", [])
+
+    if stream:
+        height = stream[0]["height"]
+
+        if height in Quality._value2member_map_:
+            return Quality(height)
```

Comment:
Very big nitpick, not necessary to change but this will break for vertical content.

![anime-kızı-anime](https://github.com/mov-cli/mov-cli/assets/66202304/03badc71-90ae-443a-a3ac-101ae548ef83)